src/ui/MessageContent/index.tsx

```diff
@@ -108,7 +108,8 @@ export default function MessageContent({
 }: Props): ReactElement {
   const messageTypes = getUIKitMessageTypes();
   const { dateLocale } = useLocalization();
-  const { config } = useSendbirdStateContext?.() || {};
+  const { config, eventHandlers } = useSendbirdStateContext?.() || {};
```

Comment:
This part is also a UI component, but since we're already using **useSendbirdStateContext**, let's skip it for now and make the changes later during refactoring ✨

Reply:
oh true. Let's keep this in mind.